Makefile

```diff
@@ -122,21 +116,21 @@ helm-generate:
 format: setup
 	@$(GOIMPORTS) -w -l $(FILES)
 
-# Adds a new dependency to glide.yaml, glide.lock and to the vendor directory
-# Usage: make vendor-get <pkg>
-# Example: make vendor-get pkg=github.com/foo/bar
-.PHONY: vendor-get
-vendor-get: setup
-	@$(GLIDE) get $(pkg) --strip-vendor
-
-# Updates all dependencies in vendor directory
-# Usage: make vendor-update
-.PHONY: vendor-update
-vendor-update: setup
-	@$(GLIDE) update --strip-vendor
-
 # Runs unit tests
 # Usage: make test
 .PHONY: test
 test: build
 	go test -v ./...
+
+#
+# go mod targets
+#
+# Usage: make add-dep pkg=istio.io/istio@1.0.4
```

Comment:
Mind changing the comment template here? The rest of the file uses the following one.

```
# Single line description of the command
# Usage: make command <args>
```